packages/optimistic-oracle/index.js

```diff
@@ -52,13 +48,29 @@ async function run({ logger, web3, pollingDelay, errorRetries, errorRetriesTimeo
     );
     const gasEstimator = new GasEstimator(logger);
 
+    // Construct default price feed config passed to all pricefeeds constructed by the keeper.
+    // The keeper needs to query prices for any identifier approved to use the Optimistic Oracle,
+    // so a new pricefeed is constructed for each identifier. This `defaultPriceFeedConfig` contains
+    // properties that are shared across all of these new pricefeeds.
+    const defaultPriceFeedConfig = {
+      lookback: 7200 // Should we pass in this object and/or its properties as input into the `run()` method?
+    };
+    const ooKeeper = new OptimisticOracleKeeper({
+      logger,
+      optimisticOracleClient: ooClient,
```

Comment:
nit: might be cleaner to just name the local var `optimisticOracleClient` and then you don't need to name it like this in the params.